src/libpriv/rpmostree-core.c

```diff
@@ -2214,6 +2213,25 @@ rpmostree_context_assemble_commit (RpmOstreeContext      *self,
     }
 
   rpmostree_output_task_end ("done");
+
+  if (!rpmostree_rootfs_prepare_links (tmprootfs_dfd, cancellable, error))
+    goto out;
+
+  if (!noscripts)
+    {
+      for (i = 0; i < n_rpmts_elements; i++)
+        {
+          rpmte te = rpmtsElement (ordering_ts, i);
+          const char *tekey = rpmteKey (te);
+          HifPackage *pkg = g_hash_table_lookup (nevra_to_pkg, tekey);
+
+          if (!run_posttrans_sync (tmp_metadata_dfd, tmprootfs_dfd, pkg,
+                                   self->ignore_scripts,
+                                   cancellable, error))
+            goto out;
+        }
+    }
+
   rpmostree_output_percent_progress_end ();
```

Comment:
Hmm, not sure what that's doing there. We should remove this, since we weren't doing any progress (I think this comes from when I used to show progress for overlaying but switched it to being a task).